source/tests/system/nidigital_driver_api_tests.cpp

```diff
@@ -168,6 +168,7 @@ class NiDigitalDriverApiTest : public ::testing::Test {
 
   bool get_bool_attribute(const char* channel_name, digital::NiDigitalAttribute attribute)
   {
+    /*
```

Comment:
Can you undo these changes? I think the tests should be good now since all the functions are there.